src/lib/remark-plugins/rewrite-tutorial-links/index.ts

```diff
@@ -83,106 +43,47 @@ function handleRewriteTutorialsLink(node: Link | Definition) {
 	node.url = rewriteTutorialsLink(node.url, TUTORIAL_MAP)
 }
 
-const getIsUrlAmbiguous = ({
-	isCollectionPath,
-	isDocsPath,
-	isProductHubPath,
-	isTutorialPath,
-}: Record<string, boolean>) => {
-	const truthyConditions = [
-		isCollectionPath,
-		isDocsPath,
-		isProductHubPath,
-		isTutorialPath,
-	].filter((condition: boolean) => condition)
-	return truthyConditions.length > 1
-}
-
 export function rewriteTutorialsLink(
 	url: string,
 	tutorialMap: Record<string, string>
 ): string {
-	let newUrl = url
+	let newUrl
 
 	try {
-		// return early if non tutorial or collection link
-		if (!learnLink.test(url) && !docsLink.test(url)) {
-			return newUrl
+		const urlObject = new URL(url, 'https://learn.hashicorp.com')
+
+		const isLearnLink = getIsLearnLink(url)
+		const isExternalDocsLink = getIsExternalDocsLink(url)
+
+		/**
+		 * Don't do anything if the link is ambiguous.
+		 */
+		if (
+			(isLearnLink === true && isExternalDocsLink === true) ||
+			(isLearnLink === false && isExternalDocsLink === false)
```

Comment:
(Chatting with Kendall over Zoom) Maybe we don't need to worry about the case where a link is neither a learn nor a doc link? We just ignore it?

Reply:
Actually, we already know we don't care about links that aren't learn or docs links! I just forgot about this code from the diff 😎 

```typescript
// return early if non tutorial or collection link
if (!learnLink.test(url) && !docsLink.test(url)) {
  return newUrl
}
```